voorhoede/head-start
# feature: Locale Selector persistence

# Changes

- Persist locale selected with Locale Selector using a cookie, and reuse it when a user revisits the root url (`/`).

# Associated issue

N/A

# How to test

1. Open preview link
2. Remember the locale ("A") you are redirected to (like `/en/`)
3. Switch to another locale ("B") using the Locale Selector (like `/nl/`)
4. Open the preview link again (`/`)
5. Verify you are redirected to locale B rather than A, meaning your locale selection is persisted and reused.


# Checklist

- ~~I have added/updated tests to prove that my feature works (if not possible please explain why)~~
- ~~I have made changes to the README and if the change affects the project setup (npm commands changed, new service added, environmental variable added)~~
- ~~I have added a decision log entry if the change affects the architecture or changes a significant technology~~
